packages/now-next/src/index.ts

```diff
@@ -226,6 +227,7 @@ export const build = async ({
 
   // Limit for max size each lambda can be, 50 MB if no custom limit
   const lambdaCompressedByteLimit = config.maxLambdaSize || 50 * 1000 * 1000;
+  const baseDir = repoRootPath || '/'; // should fallback to workPath
```

Comment:
I'm using `/` to see if any tests fail and it seems like they do so there must be some relative paths that are still wrong. @Timer @ijjk